packages/editor-ui/src/components/Sticky.vue

```diff
@@ -146,6 +148,37 @@ export default mixins(externalHooks, nodeBase, nodeHelpers, workflowHelpers).ext
 				this.$store.commit('setActiveNode', null);
 			}
 		},
+		onClick(event: Event) {
+			// Catching all sticky note clicks so we can control
+			// what happens based on the clicked element
+			const clickedElement = event.target as Element;
+			let clickedURL = '';
+
+			// Currently, tracking telemetry events on link clicks:
+			// 1. Clicks on <a> elements
+			if(clickedElement instanceof HTMLAnchorElement) {
+				clickedURL = clickedElement.getAttribute('href') || '';
+			}
+			// 2. Clicks on elements inside links
+			//    (In markdown, these should only be <img>)
+			if(clickedElement.matches('a *')) {
+				const parentLink = clickedElement.closest('a');
+				if(parentLink) {
+					clickedURL = parentLink.getAttribute('href') || '';
+				}
+			}
```

Comment:
can we push this into the markdown component component instead?  I feel like this is cleaner and more reusable...
`<n8n-markdown @link-click=`

Reply:
Sure, I'll move `@click` hander to the `Markdown` component and emit it to the `Sticky`.